mlflow/metrics/genai/prompts/v1.py

```diff
@@ -332,13 +332,7 @@ class AnswerRelevanceMetric:
         "point of the question, which is the relationship between MLflow and Databricks. "
         "Therefore, while the output is not completely irrelevant, it does not fully answer the "
         "question, leading to a lower score.",
-        grading_context={
-            "context": "MLflow is an open-source platform for managing the end-to-end machine "
-            "learning (ML) lifecycle. It was developed by Databricks, a company that specializes "
-            "in big data and machine learning solutions. MLflow is designed to address the "
-            "challenges that data scientists and machine learning engineers face when developing, "
-            "training, and deploying machine learning models."
-        },
+        grading_context={},
```

Comment:
I think this field should be optional. So lets remove it rather than it being empty